### PR TITLE
[Transport] Add transport option to CanHelp and Necessitous

### PR DESCRIPTION
**Pull request by szymon-konieczny**
Add transport option to supplies lists in CanHelp and Necessitous

### CODE REVIEW

File: src/views/CanHelp/Outlet.vue [hunk 1]
```diff
@@ -75,6 +67,12 @@
                                         Materiały do szycia:
                                         <strong>tak</strong>
                                     </div>
+
+                                    <div>
+                                        Transport:
+                                        <strong>tak</strong>
+                                    </div>
```
Comment (marckraw) [inline src/views/CanHelp/Outlet.vue]: Is transport always there ? shouldn't it be 
```
<div v-if="item.transport">
  Transport: 
  <strong>tak</strong>
</div>
```

Comment (marckraw) [reply src/views/CanHelp/Outlet.vue]: or maybe it's because backend is not yet ready, and is not returning it ?

**Review by marckraw — COMMENTED** [review]
looks ok to me, except one comment

File: src/views/CanHelp/Outlet.vue (hunk 1 above)
Comment (szymon-konieczny) [reply src/views/CanHelp/Outlet.vue]: You're right, I've missed it.

File: src/modules/Supply/components/Transport.vue [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  modifyDesc(text: string) {
+    this.rxStore.action$.next(
+      Actions.MODIFY_SUPPLY_TYPE_DESC({ listId: this.suppliesListId, text, brand: "Transport" })
+    );
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+.usage-type {
```
Comment (grzegorz-bielski) [inline src/modules/Supply/components/Transport.vue]: I think this is not needed 👀 . Need to check

**Review by grzegorz-bielski — APPROVED** [review]

**Pull request merged**